src/components/pages/team/hero/team.jsx

```diff
@@ -7,6 +7,7 @@ import { toast } from 'react-toastify';
 import { useDebouncedCallback } from 'use-debounce';
 
 import GitHubIcon from '../../../../icons/github.inline.svg';
+import MergedPRs from './merged-prs';
```

Comment:
`Eslint error`: Import order

Reply:
What should be the correct order?
Parents before siblings .. isn't this correct?
![image](https://user-images.githubusercontent.com/39879720/195503176-4f001016-868e-4c79-82db-51f194cf9615.png)